devnet/substrate_alice.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM parity/polkadot:v0.9.10 as polkadot
+FROM golang:1.17
+
+ARG CHAIN=cross-client
+ARG VERSION=v0.9.10
+ARG DD_API_KEY=somekey
+ARG METRICS_NAMESPACE=substrate.local.devnet
+ARG NODE_KEY="93ce444331ced4d2f7bfb8296267544e20c2591dbf310c7ea3af672f2879cf8f"
+
+ENV CHAIN=${CHAIN}
+ENV NODE_KEY=${NODE_KEY}
+ENV DD_API_KEY=${DD_API_KEY}
```

Comment:
Do we need to persist all these env in the final image? Or is it just used at build time?

Reply:
I think that `DD_API_KEY` is the only one we need to keep